File: cs50x/c/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int indent, int brick);

int main(void)
{
    int n;
    do
    {
        n = get_int("Input number: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        int indent = n - i - 1;
        int brick = n - indent;
        print_row(indent, brick);
    }
}

void print_row(int indent, int brick)
{
    for (int j = 0; j < indent; j++)
    {
        printf(" ");
    }
    for (int j = 0; j < brick; j++)
    {
        printf("#");
    }
    printf("  ");
    for (int j = 0; j < brick; j++)
    {
        printf("#");
    }
    printf("\n");
}

File: cs50x/week6/sentimental-mario-more/mario.py
# TODO

def main():
    height = get_int("Height: ")
    for h in range(height):
        brick = '#' * (h + 1)
        print(' ' * (height - (h + 1)) + brick + "  " + brick)


def get_int(prompt):
    height = 0
    while True:
        height = input("Height: ")
        try:
            height = int(height)
            if height in range(1, 9):
                return height
        except ValueError:
            print(f"{height} is not number")


if __name__ == "__main__":
    main()